Description: Fixes compilation error of panelkadu module
 When compiling with "-O2" there's a compilation warining regarding aliasing.
 Because "-Werror" flag is used for compilation, the warning becomes error.
 This patch disables strict-aliasing to fix compilation warning/error.
Author: Patryk Cisek <patryk@debian.org>
---
diff --git a/externalmodules/panelkadu/CMakeLists.txt b/externalmodules/panelkadu/CMakeLists.txt
index e721929..43faa81 100644
--- a/externalmodules/panelkadu/CMakeLists.txt
+++ b/externalmodules/panelkadu/CMakeLists.txt
@@ -14,6 +14,8 @@ set( CONFIGURATION_FILES
 	configuration/panelkadu.ui
 )
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-strict-aliasing")
+
 kadu_module( panelkadu
 	MODULE_SOURCES ${SOURCES}
 	MODULE_MOC_SOURCES ${MOC_SOURCES}
